src/main.rs

```diff
@@ -21,35 +21,54 @@ impl Dish {
 const TAKEAWAY_FEE: u32 = 1;
 
 #[derive(Debug, Clone)]
-struct Order; // TODO
+struct Order {
+    // tc, tofu, fr
```

Comment:
krotka do takiego zastosowania się paskudnie nie nadaje, fakt że trzeba pisać komentarz w tym miejscu to obrazuje